### ISSUE TITLE
Improving spawn dialog

### ISSUE THREAD

**Issue by morvael** [seastan/dragncards#108]
Would be nice if it would be possible to make the dialog wider (or it could be wider by default or could auto-size), provide custom labels for colum names, and allow filtering by columns other than name:
![image](https://github.com/user-attachments/assets/18d9550b-b9fd-4e40-a866-3d7dae487da9)

**Comment by seastan** [comment]
This window is meant to be simple. I only use name and packName. People can always use the builder for advanced card spawning.

**Comment by morvael** [comment]
Not sure what builder is, but this window is generally ok in functionality, just not the presentation. It should display friendly labels instead of technical tsv column names and grow wider not to have to break text, and certainly not to let one text touch another.

**Comment by morvael** [comment]
Filtering by other columns would be a bonus, but improving presentation is IMHO important.

**Comment by seastan** [comment]
I have made the UI improvements, but I'm not planning on adding column filtering at this time

**Comment by morvael** [comment]
Any configurable names for columns?

**Comment by seastan** [comment]
Yes - if you want the columns to have labels, make sure they are well-defined in gameDef.faceProperties with a label. If they don't exist in faceProperties it will just use the raw property name.

**Comment by morvael** [comment]
Label is not yet mentioned in schema https://github.com/seastan/dragncards/wiki/JSON-Schema
Also, how to specify label for default face property like "name"?
